atm.js

```diff
@@ -1,3 +1,64 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+
+  var value =0
+  var balance
+  // var depositBalance = parseInt (balance) + parseInt( $("#depo").val())
+  // var savingbalance = $(".savingbalance").html()
+
+  $(".deposit").click(function(){
+
+    var balance = $(".checkingbalance").html()
+    var depositBalance = parseInt (balance) + parseInt( $("#depo").val())
+
+    $(".checkingbalance").html(depositBalance)
+    if (balance + ($("#depo").val()) >= 0) {
+      $("#checking.account").css("background", "rgb(108,154,116)");
+    } else {""}
```

Comment:
You don't need to include this `else` statement if it doesn't do anything. Just an `if` is fine.